x/interchainqueries/keeper/grpc_query.go

```diff
@@ -56,3 +59,20 @@ func (k Keeper) QueryResult(goCtx context.Context, request *types.QueryRegistere
 	}
 	return &types.QueryRegisteredQueryResultResponse{Result: result}, nil
 }
+
+func (k Keeper) LastRemoteHeight(goCtx context.Context, request *types.QueryLastRemoteHeight) (*types.QueryLastRemoteHeightResponse, error) {
+	req := contypes.QueryConnectionClientStateRequest{ConnectionId: request.ConnectionId}
+	r, err := k.ibcKeeper.ConnectionClientState(goCtx, &req)
+	if err != nil {
+		return nil, sdkerrors.Wrapf(types.ErrInvalidConnectionID, "Connection not found")
```

Comment:
afaik it's not recommended to start error messages with a capital letter https://github.com/golang/go/wiki/CodeReviewComments#error-strings